### PR TITLE
New rule kernel_config_modules

### PR DESCRIPTION
**Pull request by jan-cerny**
Add a rule that checks if support for kernel modules is disabled. If the system can function without support for kernel modules, the support for them should be disabled. Include the rule to ANSSI R23 as a related rule.

### CODE REVIEW

File: controls/anssi.yml [hunk 1]
```diff
@@ -579,6 +579,9 @@ controls:
     - kernel_config_hibernation
     - kernel_config_binfmt_misc
     - kernel_config_legacy_ptys
+    related_rules:
+    # This rule isn't included in any profile therefore it won't appear in the built data stream.
```
Comment (Mab879) [inline controls/anssi.yml:583]: This comment, doesn't explain the why it here and not in the rules section. Please update this comment to reflect that.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for your work on this.

However, I do have some reservations about accepting this PR.
 
If we are not including this in a profile why are creating a rule?

I'm happy to discuss this with you further offline.

**Comment by jan-cerny** [comment]
> If we are not including this in a profile why are creating a rule?

I assume we wanted to create it to provide broader context in the controls file. But, I don't like it. The rule won't be present in the built data stream, so the rule is useless.

**Comment by jan-cerny** [comment]
replaced by https://github.com/ComplianceAsCode/content/pull/11571

**Pull request closed without merging**